tests/neg/not-accessible.check

```diff
@@ -9,12 +9,12 @@
 -- [E173] Reference Error: tests/neg/not-accessible.scala:13:23 --------------------------------------------------------
 13 |    def test(a: A) = a.x // error
    |                     ^^^
-   |               value x cannot be accessed as a member of (a : foo.A) from package object not-accessible$package.
+   |                     value x cannot be accessed as a member of (a : foo.A) from package object bar.
```

Comment:
That's wrong, the `package object bar` is always an object named `package` in the package `bar`, I think this should say  "from a top-level definition in package bar"

Reply:
Updated